spec.html

```diff
@@ -11892,7 +11892,7 @@
           1. Let _tailCall_ be IsInTailPosition(_thisCall_).
           1. Return ? EvaluateDirectCall(_func_, _thisValue_, |Arguments|, _tailCall_).
         </emu-alg>
-        <p>A |CallExpression| evaluation that executes step 3.a.vi is a <dfn>direct eval</dfn>.</p>
+        <p>A |CallExpression| evaluation that executes step 3.a.vii is a <dfn>direct eval</dfn>.</p>
```

Comment:
I agree about NOTE: support, but I don't see how it would solve the most troublesome cases of non-local references to a step (eg, annex B insert after/before/in place of references).